app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = t("base_title")
+    if page_title.empty?
```

Comment:
so sánh empty? blank? nil? presen?. Ở đây nên dùng cái nào

Reply:
nil? : Sẽ chỉ trả về true khi thực sự là nil

Reply:
nil? : Sẽ chỉ trả về true khi thực sự là nil
empty?: hỉ sử dụng trên các đối tượng dạng tập hợp(collection) như Hash, Array, Set. cũng có thể sử dụng empty với String (bởi vì String là mảng các ký tự). 
blank?: Trả về True cho false, nil
present?: Phủ định của blank?
=> Với giá trị mặc định đã được đặt page_title = "", dùng empty? đã đáp ứng được việc kiểm tra liệu tham số page_title đã được truyền vào hay sẽ sử dụng giá trị mặc định cho những tính toán tiếp theo